boards/arm/adafruit_kb2040/doc/index.rst

```diff
@@ -0,0 +1,118 @@
+.. _adafruit_kb2040:
+
+Adafruit KB2040
+###############
+
+Overview
+********
+
+The Adafruit KB2040 is a small, low-cost, versatile board from
+Adafruit. It is equipped with an RP2040 SoC, an on-board RGB Neopixel,
+a USB connector, and an QT connector. The USB bootloader allows it
```

Comment:
a QT connector

Reply:
Added the full name "STEMMA QT connector" to match the adafruit docs/naming.